### DISCUSSION TITLE
Denied Data Access Renewal

### DISCUSSION THREAD

**Discussion by rt-asdf** [now-i-know-my-abcd/docs#91] [Stats & Code Help] [answered]
Hi. I am a researcher from China. Recently I tried to renew my data permission, but the NIH Help Desk replied me that "NIH has received and processed your Data Access Request (DAR). This request is denied." I am confused as there is no clarification of the reason. In previous years I would be informed the reason if the application did not meet the requirement, so that I could make revision accordingly. I am not sure if anyone else shares similar experience and wondering what to do next. Should I keep contact the NIH Help Desk for more information, or did I miss anything? Thank you in advance for any information.

**Comment by ebrahimebrahim** [comment]
In the past they have explained to me the reason for being denied and helped me to repair the problem. The NIH Help Desk has always been very responsive and helpful in my experience, so yes suggest you ask them what is needed to fix your application. There are a lot of little things that are easy to get wrong.

**Reply by rt-asdf** [reply]
Thank you for your reply! I am trying to contact the NIH Help Desk for further details. By the way, I know two researchers, also from China, received the denial recently in the same words. I'm not sure if I missed any updates regarding any restriction of usage of the ABCD data this year. Hope things will work out fine.

**Reply by Junezhujun** [reply]
Hello! Have you solved your problem? I come from Sichuan University,China.Recently, I have been following the tutorial to apply for access to the ABCD/CCF data . However, upon submitting my data access request, it was immediately denied without any explanation provided.When I reached out to the NDA Helpdesk with my query, I received the following response: "NIH has received and processed your Data Access Request (DAR). This request is denied." I would appreciate any suggestions you might have.

**Reply by rt-asdf** [reply]
Nope. Regarding my further questions the NIH Help Desk replied me the exact same words with no further explanation...

**Comment by angielaird** [comment, marked as answer]
An executive order issued by the Biden Administration now prohibits the sharing of Americans' bulk personal data, including genomic data, in certain cases. You can read about it [here](https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/#%3A~%3Atext=%28c%29%20The%20term%20“country%2CStates%20or%20the%20security%20and). 

The ABCD Study has posted announcements about this issue, as well as the current problems with data downloads, on the [ABCD Wiki splash page](https://wiki.abcdstudy.org).

**Reply by rt-asdf** [reply]
Hi Angie. Thank you for the information. It does seem unsolvable for an indefinite period... Anyway, I would like to express my appreciation to the ABCD team for maintaining such a valuable cohort. Hope there will be opportunities for collaboration in the future.
